lnrpc/signrpc/signer.proto

```diff
@@ -2,6 +2,8 @@ syntax = "proto3";
 
 package signrpc;
 
+option go_package = "github.com/lightningnetwork/lnd/lnrpc/signrpc";
```

Comment:
what's this?

Reply:
To make it work outside the gopath with modules. The dir structure doesn't need to match the package structure anymore there.